contracts/modules/SecurityManager.sol

```diff
@@ -186,6 +188,14 @@ abstract contract SecurityManager is BaseModule {
         return (config.recovery, config.executeAfter, config.guardianCount);
     }
 
+    /**
+    * @notice Clears the active session of a wallet if any.
+    * @param _wallet The target wallet.
+    */
+    function clearSession(address _wallet) public onlyWalletOwnerOrSelf(_wallet) {
```

Comment:
I find it a little inconsistent that creating session is on the `TransactionManager` and clearing a session is on the `SecurityManager`. They should be defined on the same contract.

Since the `session` object is declared on the `BaseModule` I would prefer to have the internal `_clearSession()` defined on the `BaseModule` and maintain the external `clearSession()` on the `TransactionManager`.